src/Standard.java

```diff
@@ -0,0 +1,4 @@
+
+public interface Standard<T> {
+
```

Comment:
If you're going to make your own Standard, make sure you list the three key contracts.